ASM/src/hacks.asm

```diff
@@ -1238,6 +1238,14 @@ skip_GS_BGS_text:
 .org 0xFEB000 + 0x10C0 - 0x32A090 + 0x32A158
 .word   0xDE000000, 0x09000010
 
+;==================================================================================================
+; Increase transparency of red ice crystals blocking chests in CTMC
+;==================================================================================================
+
+; Replaces addiu    t7, $zero, 0x00FF
+.orga 0xDB3244
+    j       red_ice_alpha
```

Comment:
What is the next instruction in the delay slot? Is that affected by any of the new code?

This should also be jal, not j.

Reply:
The instruction in the delay slot sets the action function and is unaffected. I added a comment with the instruction.

I used j instead of jal as a shortcut to avoid having to push the old ra to the stack. I don't think there's enough room in the function to do that properly. red_ice_alpha does the skipped instructions instead.